recipes/synspace/recipe.yaml

```
@@ -0,0 +1,51 @@
+context:
+  name: synspace
+  version: 0.3.0
+
+package:
+  name: ${{ name|lower }}
+  version: ${{ version }}
+
+source:
+  url: https://pypi.org/packages/source/${{ name[0] }}/${{ name }}/synspace-${{ version }}.tar.gz
+  sha256: d0e004041e8e9ea401b699241d8f18574d8a4893e179260316ff8e5d1ff67515
+
+build:
+  number: 0
+  noarch: python
+  script: ${{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
```

```suggestion
  script: python -m pip install . -vv
```

I think with these new recipes we don't need `${{ PYTHON }}`, which is also reflected in the example recipe for v1

---

I think it's needed for cross compilation purposes, but it won't be needed since its `noarch` so we can remove it.